robottelo/vm.py

```diff
@@ -293,6 +297,17 @@ def put(self, local_path, remote_path=None):
             )
         ssh.upload_file(local_path, remote_path, hostname=self.ip_addr)
 
+    def fetch_domain(self):
```

Comment:
Totally agree, I think we better avoid using `run` or `ssh.command` if possible.